storage/innobase/include/ut0ut.h

```diff
@@ -88,6 +88,11 @@ struct ut_when_dtor {
    the YieldProcessor macro defined in WinNT.h. It is a CPU architecture-
    independent way by using YieldProcessor. */
 #  define UT_RELAX_CPU() YieldProcessor()
+# elif defined(_AIX)
+# define UT_RELAX_CPU() do { \
+     volatile lint      volatile_var; \
+     os_compare_and_swap_lint(&volatile_var, 0, 1); \
+   } while (0)
```

Comment:
Per my understanding UT_RELAX_CPU() should be hardware specific (not OS specific). What's your underlying hardware?